sunpy/instr/aia/response.py

```diff
@@ -41,76 +42,105 @@ class description here!
 
     Parameters
     ----------
-    wave: array
-        wavelengths imaged by the instrument and centered around the channel wavelength
-    geoarea: float
-        the geometric area of the each channel: EUV = 83.0 cm**2, UV = 30.8 cm**2
-    elecperev: float
-        a conversion factor for electrons per electronic volts
-    ent_filter: array
-        size of the entire filter on the instrument
-    fp_filter: array of floats
-        size of fp filter <<<<
-    contam: array of floats
-        maps the fluctuation of the instrument
-    cross_area: array of flats
-        cross area of the instrument
-    secondary: array of floats
-        size of secondary mirror
-    primary: array
-        size of the primary mirror
-    ccd: array, floats
-        size of the ccd
-
-
-    example:
-    from sunpy.instr.aia import aia                          #Q) redundant!?
-    effarea = aia.response.aia_inst_genx_to_dict(path_to_file)
+    path_to_genx_file: string
+        give a path to a instrument file from .genx output to initialize a dataframe for the class
+    save_to_outfile: bool
+        loads channel properties into a dataframe and saves to a .csv file
+    csv: string
+        alternately, give a path to the csv file to read in data information where the keys are channel wavelength centers and the values are properties
+
+    Attributes
+    ----------
+    properties: array of strings
+        The names of properties that are imported from SSW
+    wavelength_centers: array of integers
+        The center wavelength of each wavelength range or channel imaged with AIA
+    dataframe: pandas dataframe
+        The property information stored after being imported from SSW
+    wavelength_range: array
+        the range of wavelengths imaged per channel with AIA
+
+
+    # Example:
```

Comment:
When doing example code blocks in the docstrings you should lay them out like this:
```python

Examples
------------

First example::
    >>> import sunpy

Second Example::
    >>> import astropy
```